stack.yaml

```diff
@@ -17,3 +17,4 @@ extra-deps:
 - AhoCorasick-0.0.4
 resolver: lts-22.10
 
+system-ghc: true
```

Comment:
Does this solve a particular problem?

Reply:
Oops, I have that in my personal `stack.yaml` as a workaround for some bug or other (I forget what), I didn't mean to commit it.  I'll revert that change.